### PR TITLE
onotes bootstrap convertion

### PR DESCRIPTION
**Pull request by robertogagliotta**
Unite testing ok
img_2258
img_2259
img_2260

Scripts are ready for review

### CODE REVIEW

**Comment by mdsupport** [comment]
Just added PR #1001 which is useful in these type of interfaces.  Depending on time/effort availability, you could just add a class to the textarea tags and enrich the content including ability to incorporate images etc.
Better would be eliminating the edit link and letting users makes changes inline - really cool way for users familiar with word editors.

Caution: Adding html code would entail display adjustments wherever used.

**Comment by robertogagliotta** [comment]
I will try ckeditor add-on and see how it does. Thanks mdssupport

File: interface/main/onotes/office_comments.php [hunk 1]
```diff
@@ -72,26 +71,22 @@
...
-            $card .= '    <div class="panel-body">';
-            $card .=          nl2br(text($note->getBody()));
-            $card .= '    </div>';
-            $card .= '</div>';
-
+            $card .= '</th></thead>';
+            $card .= '<tr><td>';
+            $card .= nl2br(text($note->getBody()));
+            $card .= '</th></td>';
+            $card .= '';
```
Comment (bradymiller) [inline interface/main/onotes/office_comments.php]: Were these changes necessary? (this is basically removing bootstrap grid and going back to using table for layouts, which we are avoiding)

File: interface/main/onotes/office_comments.php [hunk 2]
```diff
@@ -32,18 +25,20 @@
...
-<br>
-
-<table border=0 width=100%>
+<div class="container">
+   
+   <div class="row">
+      <div class="col-sm-6">
+         <div class="form-group form-horizontal page-header">
+            <h3><label class='col-sm-4 text-right'><?php echo xlt('Office Notes'); ?></label></h3>
+            <p class="more control-label col-sm-1"><a href="office_comments_full.php" onclick='top.restoreSession()'><?php echo text($tmore); ?></a></p>
```
Comment (bradymiller) [inline interface/main/onotes/office_comments.php]: turn this link into a button

File: interface/main/onotes/office_comments.php [hunk 3]
```diff
@@ -2,20 +2,13 @@
...
- *
- * @package OpenEMR
- * @author  Brady Miller <brady.g.miller@gmail.com>
- * @link    http://www.open-emr.org
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @author    Roberto Vasquez <robertogagliotta@gmail.com>
+ * @Copyright (C) 2011-2017  Brady Miller <brady.g.miller@gmail.com>
+ * @Copyright (C) 2017 Roberto Vasquez <robertogagliotta@gmail.com>
```
Comment (bradymiller) [inline interface/main/onotes/office_comments.php]: change to
``````
 * @copyright       Copyright (C) 2011-2017  Brady Miller <brady.g.miller@gmail.com>
 * @copyright       Copyright (C) 2017 Roberto Vasquez <robertogagliotta@gmail.com>

File: interface/main/onotes/office_comments_full.php [hunk 4]
```diff
@@ -2,20 +2,13 @@
...
- *
- * @package OpenEMR
- * @author  Brady Miller <brady.g.miller@gmail.com>
- * @link    http://www.open-emr.org
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @author    Roberto Vasquez <robertogagliotta@gmail.com>
+ * @copyright (C) 2017 Brady Miller <brady.g.miller@gmail.com>
+ * @Copyright (C) 2017 Roberto Vasquez <robertogagliotta@gmail.com>
```
Comment (bradymiller) [inline interface/main/onotes/office_comments_full.php]: * @copyright       Copyright (C) 2017  Brady Miller <brady.g.miller@gmail.com>
 * @copyright       Copyright (C) 2017 Roberto Vasquez <robertogagliotta@gmail.com>

File: interface/main/onotes/office_comments_full.php [hunk 5]
```diff
@@ -70,18 +76,22 @@
...
-<a href="office_comments.php" onclick='top.restoreSession()'>
-
-<span class="title"><?php echo xlt('Office Notes'); ?></span>
-<span class="back"><?php echo text($tback); ?></span></a>
-
+   <div class="row">
+      <div class="col-sm-6">
+         <div class="form-group form-horizontal page-header">
+            <h3><label class='col-sm-4 text-right'><?php echo xlt('Office Notes'); ?></label></h3>
+            <p class="back control-label col-sm-1"><a href="office_comments.php" onclick='top.restoreSession()'><?php echo text($tback); ?></a></p>
```
Comment (bradymiller) [inline interface/main/onotes/office_comments_full.php]: this should be a button

File: interface/main/onotes/office_comments_full.php [hunk 6]
```diff
@@ -2,20 +2,13 @@
 /**
  * Viewing and modification/creation of office notes.
```
Comment (bradymiller) [inline interface/main/onotes/office_comments_full.php]: all the buttons in this script should be using the new bootstrap buttons

**Comment by bradymiller** [comment]
hi @robertogagliotta 
left a review. this is a bit of a tough one since there are several elements going on. after you revise from this review, then we'll tackle some more finer details and get this thing done.
-brady

File: interface/main/onotes/office_comments.php (hunk 1 above)
Comment (robertogagliotta) [inline interface/main/onotes/office_comments.php]: Since they were not inside a bootstrap tag, I didn't know the panel-default  panel-heading  panel-body are bootstrap themes. I'll put them back

File: interface/main/onotes/office_comments.php (hunk 2 above)
Comment (robertogagliotta) [inline interface/main/onotes/office_comments.php]: btn-link is not in themes/core.css and I do not find a btn-more or something like that. I will us btn-show.
